config/locales/fr.yml

```diff
@@ -664,6 +664,7 @@ fr:
   are_you_sure_you_want_to_remove_this_observation_from_project: |
     Êtes-vous sûr de vouloir supprimer cette observation de %{project}?
   are_you_sure_you_want_to_remove_this_taxon_from_checklist: "Êtes-vous certain de vouloir supprimer ce taxon de la check-list de « %{place} » ?"
+  are_you_sure_you_want_to_flag_as_spammer: Êtes-vous sûr de vouloir signaler %{user} comme spammeur ?
```

Comment:
Please don't add translations. We manage translations through https://crowdin.com/project/inaturalistweb, so the text will be translated by volunteers once the source strings from en.yml are merged into the main branch.